helpers.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, stdout, stderr, exited):
         self.stderr = stderr
         self.exited = exited
         self.return_code = exited
+        self.failed = exited != 0
```

Comment:
Smart ;)

While this works for today's code, where we only create a `Result` object once to return to the user, it has a pitfall if the user changes the `exited` value.

If I do `my_result = Result(... exited=0)` and then `my_result.exited = 1`, `failed` will have an stale value.

The Python way of doing this is with a `@property` decorator :nerd_face:

Reply:
Ah, that's a nice way of doing it